bluetooth/bluetooth-control

```diff
@@ -0,0 +1,160 @@
+#!/usr/bin/env bash
+
+# Set up variables
+VOLUME_UP_GPIO=$(</usr/src/volume_up_gpio)
+
+BLUETOOTH_GPIO=$(</usr/src/bluetooth_gpio)
+
+VOLUME_DOWN_GPIO=$(</usr/src/volume_down_gpio)
+
+NEXT_GPIO=$(</usr/src/next_gpio)
+
+PREVIOUS_GPIO=$(</usr/src/previous_gpio)
+
+OUTPUT_GPIO=$(</usr/src/output_gpio)
```

Comment:
As I understand it, the other script writes the values into these files, and then this scripts reads the values. 
Are the files used anywhere else? Can't we just have all done here and avoid the files alltogether?